src/lwc/geFormService/geFormService.js

```diff
@@ -109,7 +113,14 @@ class GeFormService {
      * @returns opportunityId
      */
     handleSave(sectionList) {
-        
+        let diRecord = this.getDataImportRecord(sectionList);
+
+        const opportunityID = this.createOpportunityFromDataImport(diRecord);
```

Comment:
Realize this was already present but Is "createOpportunityFromDataImport" really the best name for this method? This is really kicking off the processing of the DI and potentially creates a lot more than just the opp, if I read it correctly

Reply:
I agree that this method should probably be re-named but since it is the method used in single/SGE mode maybe we can get GEM's input.  @vmartinov-sforg or @bdvorachek or @KietTruongSFDO what do you all think about renaming this `createOpportunityFromDataImport` method?

Reply:
No objections to renaming it. It should probably just be renamed to 'saveAndProcessGift' since that's the import name it's calling? Or ideally 'saveAndProcessSingleGift' which is the backend apex method name and rename the import so that it's consistent.